### PR TITLE
docs: changelog-2024-07-12

### PR DESCRIPTION
**Pull request by danieltprice**
https://neon-next-git-docs-changelog-07-12-2024-neondatabase.vercel.app/docs/changelog

### CODE REVIEW

File: content/changelog/2024-07-12.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  ![added create child branch button](/docs/relnotes/branches_create_child_branch.png)
+- Easier navigation from your listed database on **Roles and databases** tab to the Drizzle Studio-based **Tables** page, where you can explore and modify your data directly. The actions kebab also includes options to delete your database, as well as navigate to the SQL Editor, and we preserve your database selection as you travel.
+  ![databases edit and new kebab](/docs/relnotes/databases_actions.png)
+- We've changed the metric displayed on the Branches from **active time** to the more helpful **compute time** metric, giving you a better sense of how much compute resources a given branch is consuming. For more details about consumption metrics, see [Usage metrics](/docs/introduction/usage-metrics).
+
+### Even more protection
+
+Last week, we introduced protection against accidentally deleting protected branches. This week, we've added more safeguards:
+
+- **Default Branches:** We've made it so you can't accidentally delete your default branch. If you need to delete that branch, make another branch the default first.
```
Comment (danieltprice) [inline content/changelog/2024-07-12.md:39]: "We've made it so you can't accidentally delete your default branch." - this was true before, so this item can be removed.

File: content/changelog/2024-07-12.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+- **Bootstrap new applications with Neon using the `create-app` command**
+
+  You can now scaffold applications using the CLI with the `create-app` command. Choose your package manager, framework, ORM, and authentication method, and go. This exciting new feature is just getting started. More to come!
+
+### Added polish to our Branches page
+
+Branching is still a relatively new concept in the database world, and we want to make your experience with it as intuitive as possible. Here are our latest updates to the **Branches** page:
+
+- Made it easier to create child branches by adding a clear-as-day **Create child branch** button to the detailed Branches view. It's still an option under the Actions dropdown, but now you've got the can't miss button too.
```
Comment (danieltprice) [inline content/changelog/2024-07-12.md:29]:
```suggestion
- Made it easier to create child branches by adding a clear-as-day **Create child branch** button to the detailed Branches view. It's still an option under the Actions dropdown, but now you've got the can't-miss button too.
```

File: content/changelog/2024-07-12.md (hunk 1 above)
Comment (bgrenon) [reply content/changelog/2024-07-12.md]: done

File: content/changelog/2024-07-12.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+Last week, we introduced protection against accidentally deleting protected branches. This week, we've added more safeguards:
+
+- **Default Branches:** We've made it so you can't accidentally delete your default branch. If you need to delete that branch, make another branch the default first.
+- **Resetting Protected Branches:** We now prevent you from using **Reset from parent** on protected branches (often, this is your production branch) unless you unprotect the branch first.
+
+### Fixes & improvements
+
+- Fixed a misleading item in our Quickstart, where we told you that your compute had already started while it was still in transition. Trying to connect in that state would fail. Now, when you see your compute has started, it is definitely started. You can grab your connection string and go.
+- Changed `ip-allow list` to include a column letting you know if the list is restricted to protected branches only. Previously, this column showed `primary-branch only` instead.
```
[the remark this answers is not in the document]
Comment (bgrenon) [reply content/changelog/2024-07-12.md]: Revised.

**Review by bgrenon — APPROVED** [review]

**Pull request merged**